bft/dialects/types.py

```diff
@@ -83,7 +83,7 @@ class DialectFile(NamedTuple):
     scalar_functions: List[DialectFunction]
     aggregate_functions: List[DialectFunction]
     uri_to_func_prefix: Dict[str, str]
-    supported_types: List[str]
+    supported_types: Dict[str, str]
```

Comment:
Given that the key is made up of a key and a value itself is this type more like a tuple or dict?

Reply:
changed the value to dict